channeldb/db_test.go

```diff
@@ -33,3 +33,37 @@ func TestOpenWithCreate(t *testing.T) {
 		t.Fatalf("channeldb failed to create data directory")
 	}
 }
+
+func TestWipe(t *testing.T) {
+	t.Parallel()
+
+	// First, create a temporary directory to be used for the duration of
+	// this test.
+	tempDirName, err := ioutil.TempDir("", "channeldb")
+	if err != nil {
+		t.Fatalf("unable to create temp dir: %v", err)
+	}
+	defer os.RemoveAll(tempDirName)
+
+	// Next, open thereby creating channeldb for the first time.
+	dbPath := filepath.Join(tempDirName, "cdb")
+	cdb, err := Open(dbPath)
+	if err != nil {
+		t.Fatalf("unable to create channeldb: %v", err)
+	}
+	defer cdb.Close()
+
+	if err := cdb.Wipe(); err != nil {
+		t.Fatalf("unable to wipe channeldb: %v", err)
+	}
+	// check errors are returned
+	// increase cover
+	_, err = cdb.FetchAllOpenChannels()
+	if err != ErrNoActiveChannels {
+		t.Fatalf("fetching open channels and expecting error: '%v' got: '%v'", ErrNoActiveChannels, err)
```

Comment:
lines here and below are too long. Should be wrapped at 80 chars with 8 char tab width.